tests/system/test_system.py

```diff
@@ -2333,7 +2513,7 @@ def test_execute_sql_returning_transfinite_floats(self):
                 )
             )
             self.assertEqual(len(rows), 1)
-            float_array, = rows[0]
+            (float_array,) = rows[0]
```

Comment:
Just curious: why `()` is required?

Reply:
It's not, I think my auto linter added it for some reason. removed

Reply:
So why don't we write: 

```py3
float_array = rows[0][0]
```

The above way is still confusing, which is usually used for tuples.

Reply:
> ... nevermind. the system tests fail if this isn't there.

Interesting. I haven't had any issues with this line before. However, I do agree it is a confusing way to write this.

Reply:
fixed, went with rows[0][0]